packages/migrate/src/helpers/typescript-ast.ts

```diff
@@ -92,3 +92,25 @@ export function isSourceCodeChanged(originalText: string, updateText: string): b
   // Compares source codes without spaces.
   return originalText.replace(/\s+/g, ' ') !== updateText.replace(/\s+/g, ' ');
 }
+
+export function insertStringIntoSourceFile(
+  sourceFile: ts.SourceFile,
+  insertAt: number,
+  insertedStr: string
+): string {
+  const originalText = sourceFile.getFullText();
+  const newText = `${originalText.substring(0, insertAt)}${insertedStr}${originalText.substring(
+    insertAt
+  )}`;
+  const newSourceFile = ts.updateSourceFile(
+    sourceFile,
+    newText,
+    ts.createTextChangeRange({ start: 0, length: originalText.length }, newText.length)
+  );
+  return ts
```

Comment:
Seems you don't need to work with SourceFile here. 
Can this function be just 
```
insertStringAtPosition(originalText, insertAt, insertedStr) {
    return `${originalText.substring(0, insertAt)}${insertedStr}${originalText.substring(insertAt)}`;
}
```
?

Reply:
Removed printer